README.md

```diff
@@ -97,15 +110,190 @@ A quick look at the top-level files and directories you'll see in a Gatsby proje
+[data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+```
+
+The `@apply` directive exists because we are using [Tailwind css](https://tailwindcss.com/). To learn more about tailwind, [see their docs](https://tailwindcss.com/docs). To know more about our best practices on using tailwid, see the `Styling Components` section.
+This `data-store-button` is a css data attribute selector. To know which selectors are available, check [store-ui docs](https://storeui.netlify.app/)
+
+Now, open `Button.tsx` and import this CSS with:
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {}
+
+function Button (props: Props) {
+  return <UIButton {...props} />
+}
+
+export default Button
+```
+
+For most components you would stop here. However, buttons can have different variants. For instance, suppose you want to have a button component with primary and muted variants. To add variants to the component, you can, on `Button.tsx`
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {
+  variant: 'muted' | 'primary'
+}
+
+function Button ({variant, ...props}: Props) {
+  return <UIButton className={variant} {...props} />
+}
+
+export default Button
+```
+
+and than, on `Button.module.css`:
+```css
+.primary [data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+
+.muted [data-store-button]: {
+  @apply p-0 bg-muted-100;
+}
+```
+
+Now we have a styled Button component that accepts different variants!! 🎉
+
+The aforementioned guide works well for UI components. However, sections are more complex and usually don't have variants, since they usually serve a single responsibility in the page. For sections, you can use tailwind natively like:
+
+```tsx
+// components/sections/MySection/MySection.tsx
+...
+function MySection {
+  return (
+    <>
+      <div className="p-2 flex items-center">
+        <p className="h-4 p-5 mx-2">Hello World</p>
+        <Button variant="muted">
+      </div>
+    </>
+  )
+}
+...
+```
+
+## 🖊️ Styling Components
+This starter uses [Tailwind CSS](https://tailwindcss.com/) for styling. If you want, you can remove it and use other solutions. Both Gatsby and Store UI support many different CSS frameworks, like [emotion](https://emotion.sh/docs/introduction), [stitches](https://stitches.dev/docs/introduction) or even no CSS framework at all. 
+This guide covers best practices and patterns to use when styling with Tailwind.
+
+Tailwind is a utility-first CSS framework. The goal of this starter is to make it possible for developers to use Tailwind themes to change the look of the store. To accomplish this, a few things needs to be respected:
+
+1. Do never use hard coded colors/padings/spacings etc. e.g.: `px-[10px], bg-[#fff]`
```

Comment:
```suggestion
1. Do never use hard coded colors/padings/spacings etc, e.g., `px-[10px]`, `bg-[#fff]`.
```

Reply:
I think it would be good to add what they are supposed to do instead (use aliases?)